src/alpine/3.6-WithNode/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6
+
+RUN apk update
+
+ENV NODE_VERSION 10.13.0
+
+### BUILD NODE
+RUN apk add --no-cache \
+        libstdc++ \
+    && apk add --no-cache --virtual .build-deps \
+        binutils-gold \
+        curl \
+        g++ \
+        gnupg \
+        libgcc \
+        make \
+        # gpg keys listed at https://github.com/nodejs/node#release-team
+        && for key in \
+            94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+            FD3A5288F042B6850C66B31F09FE44734EB7990E \
+            71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+            DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+            C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+            B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+            56730D5401028683275BD23C23EFEFE93C4CFFFE \
+            77984A986EBC2AA786BC0F66B01FBB92821C587A \
+            8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
+        ; do \
+            gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
```

Comment:
It seems inconsistent that the `||` operator is at the end of the lines here but the `&&` operators are at the beginning.  Most often in Dockerfiles the logical operator is at the beginning of the line.